docs/guide/extending_page.md

```diff
@@ -106,6 +106,144 @@
+        default:
+            field_templates:
+                - { template: 'AppBundle:ezform_field.html.twig', priority: 0 }
+    ```
+    
+    At this point, the attribute type configuration is complete, but it requires a mapper.
+    Depending on the complexity of the type, you can use a `GenericFormTypeMapper` or create your own.
+    
+    For a generic mapper, register it as a service by adding:
+    
+    ``` yaml
+     my_application.block.attribute.my_string:
+            class: EzSystems\EzPlatformPageFieldType\FieldType\Page\Block\Attribute\FormTypeMapper\GenericFormTypeMapper
+            arguments:
+                $formTypeClass: AppBundle\Block\Attribute\MyStringAttributeType
+            tags:
+                - { name: ezplatform.page_builder.attribute_form_type_mapper, alias: my_string }
+    ```
+    
+    For creating your own mapper, create a class that inherits after `EzSystems\EzPlatformPageFieldType\FieldType\Page\Block\Attribute\FormTypeMapper\AttributeFormTypeMapperInterface`.
+    Then, register the class along with a tag by creating a `src/AppBudle/Block/Attribute/MyStringAttributeMapper.php` file:
+    
+    ``` php
+    <?php declare(strict_types=1);
+    
+    namespace AppBundle\Block\Attribute;
+    
+    use EzSystems\EzPlatformPageFieldType\FieldType\Page\Block\Definition\BlockAttributeDefinition;
+    use EzSystems\EzPlatformPageFieldType\FieldType\Page\Block\Definition\BlockDefinition;
+    use Symfony\Component\Form\FormBuilderInterface;
+    
+    class MyStringAttributeMapper implements AttributeFormTypeMapperInterface
+    {
+        /**
+         * @param \Symfony\Component\Form\FormBuilderInterface $formBuilder
+         * @param \EzSystems\EzPlatformPageFieldType\FieldType\Page\Block\Definition\BlockDefinition $blockDefinition
+         * @param \EzSystems\EzPlatformPageFieldType\FieldType\Page\Block\Definition\BlockAttributeDefinition $blockAttributeDefinition
+         * @param array $constraints
+         *
+         * @return \Symfony\Component\Form\FormBuilderInterface
+         * @throws \Exception
+         */
+        public function map(
+            FormBuilderInterface $formBuilder,
+            BlockDefinition $blockDefinition,
+            BlockAttributeDefinition $blockAttributeDefinition,
+            array $constraints = []
+        ): FormBuilderInterface {
+            return $formBuilder->create(
+                'value',
+                MyStringAttributeType::class,
+                [
+                    'constraints' => $constraints,
+                ]
+            );
+        }
+    }
+    ```
+    
+    The final step is to register your mapper as a service:
+    
+    ``` yaml
+    AppBundle\Block\Attribute\MyStringAttributeMapper:
+            tags:
+                - { name: ezplatform.page_builder.attribute_form_type_mapper, alias: my_string }
+    ```
+    
+    Now, create a block containing your custom attribute:
+    
+    ``` yaml hl_lines="8 9 10 11"
+    blocks:
```

Comment:
Let's add `ezplatform_page_fieldtype:` before, to be consistent with earlier examples